main/lsp/notifications/did_change_configuration.cc

```diff
@@ -0,0 +1,22 @@
+#include "main/lsp/notifications/did_change_configuration.h"
+#include "main/lsp/LSPPreprocessor.h"
+
+namespace sorbet::realmain::lsp {
+DidChangeConfigurationTask::DidChangeConfigurationTask(const LSPConfiguration &config,
+                                                       std::unique_ptr<DidChangeConfigurationParams> params)
+    : LSPTask(config, LSPMethod::WorkspaceDidChangeConfiguration), params(move(params)) {}
+
+LSPTask::Phase DidChangeConfigurationTask::finalPhase() const {
+    return LSPTask::Phase::RUN;
+}
+
+void DidChangeConfigurationTask::preprocess(LSPPreprocessor &preprocessor) {
+    preprocessor.resume();
```

Comment:
ooc, what's up with this? it's not a part of sorbet i'm familiar with

Reply:
my _guess_ is that we *do not* want this. the only other place i see this being called is in the sorbet_resume notification, which is a test-only method that pairs with sorbet_pause that makes it easier to test multi-threaded code

Reply:
yeah this was lurked in from copy-pasta